pkg/syncer/storagepool/diskDecommissionController.go

```diff
@@ -0,0 +1,277 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Can you correct the license format. I think there are some new lines in between.  Refer to some existing files.

Reply:
done